bindata/network/multus/multus.yaml

```diff
@@ -348,6 +348,7 @@ spec:
             memory: 150Mi
         securityContext:
           privileged: true
+        terminationGracePeriodSeconds: 10
```

Comment:
`Without setting the gracePeriodSeconds lower, a pod in the Multus daemonset can take ~35 seconds to exit.`
So what was the previous default value ?

Reply:
the default value is apparently 30s. So give or take some API delays, it may take ~35 seconds?

Reply:
Thanks Surya, I think the default for `terminationGracePeriodSeconds` is 30 seconds. So I think we get somewhere in the neighborhood of 20 seconds faster to delete a pod in this daemonset. Independent of this PR we're also looking at trapping SIGTERM more efficiently which I think will be even more efficient (but it'll be done in the entrypoint for the daemonset).

In general, it hasn't "been a huge problem" but since `maxUnavailable` defaults 1, only one pod will go down in a cluster, so.... If you've got a cluster with 200 nodes, it's takes a long time to upgrade, making it a.... huger problem.

Reply:
Thanks Doug for explaining it to me :) . Makes sense.